src/main/java/Task.java

```diff
@@ -0,0 +1,29 @@
+public class Task {
+    private String taskName;
+    private boolean isDone;
+
+    public Task(String taskName) {
+        this.taskName = taskName;
+        this.isDone = false;
+    }
+
+    public void setDone(boolean done) {
+        this.isDone = done;
+    }
+
+    public String getTaskName() {

```

Comment:
Seems like this function is unnecessary due to the use of toString to print the tasks